control/control.go

```diff
@@ -317,7 +332,7 @@ func (c *Controller) Solve(ctx context.Context, req *controlapi.SolveRequest) (*
 		Exporter:        expi,
 		CacheExporter:   cacheExporter,
 		CacheExportMode: cacheExportMode,
-	}, req.Entitlements)
+	}, req.Entitlements, srcPol)
```

Comment:
If the policy is part of `controlapi.SolveRequest`, it should probably be passed as part of `frontend.SolveRequest`?

Reply:
Decoupled from `frontend.SolveRequest` to follow the entitlement model
https://github.com/moby/buildkit/pull/2816#issuecomment-1154410535